polytope/polytope.py

```diff
@@ -291,26 +291,34 @@ def intersect(self, other, abs_tol=ABS_TOL):
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
     def translation(self, d):
-        """Translate the Polytope by a given vector.
+        """Returns a copy of C{self} translated by the vector C{d}.
 
         Consult L{polytope.polytope._translate} for implementation details.
 
         @type d: 1d array
-        @param d: The translation vector.
-
-        @rtype: L{Polytope}
-        @return: A translated copy of the Polytope.
         """
         newpoly = self.copy()
         _translate(newpoly, d)
         return newpoly
 
     def rotation(self, u, v, theta=None):
-        """Rotate the Polytope. Only simple rotations are implemented at this
+        """Returns a rotated copy of C{self}.
+
+        Use one of two methods that to describe the plane of rotation and the
+        angle of rotation (in radians) with u, v, and theta.
+
+        (1) u and v are the indicies (0, N] of two basis vectors, and theta is
```

Comment:
I think that `0..(N-1)` or `0..N-1` are correct, but `[0, N)` is ambiguous, because the indices should be integers. In a typed formalism, one could claim to have declared that `u, v` are of "integer type" (whatever that may mean). However, in `polytope` we have neither type declarations of any form, nor are integers the norm. So, I think that `0..(N-1)` avoids this ambiguity. Explicit is better than implicit (PEP 20).

In a context that involves real numbers, I prefer `[0, N)` to `[0, N[`, because I find it easier to distinguish a a closing parentheses `)` from a closing bracket `]`, than an opening bracket `[` from a closing bracket `]`. I do appreciate the symmetry of the latter notation though.

I believe that "indicies" is a typo. Both "indices" and "indexes" [appear](https://en.wiktionary.org/wiki/index) to be used as plural form of "index". A remark about the same was made [earlier](https://github.com/tulip-control/polytope/pull/39#issuecomment-275926090).

Also, I think that if one method is supported now, then No.2 should be omitted. Why have the user read two items, and then find out the 2nd isn't available?

I am opposed to the polymorphism of `u, v`. I had to read No.1 multiple times to convince myself that I understand correctly. `u, v` should better have one type in each signature. If they are integer indices, it is more familiar to name them `i, j` or `m, n`.

"basis" could be ambiguous. We could specify that the basis is the same as that used to express the coordinates.

Relevant for those interested in untypeness: `omega` is in transition from typed to untyped (cf https://github.com/johnyf/omega/issues/3).